test/integration/bundle_int_test.go

```diff
@@ -221,6 +221,13 @@ func (suite *BundleIntegrationTestSuite) TestJSON() {
 	cp.ExpectExitCode(0)
 	AssertValidJSON(suite.T(), cp)
 
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/Bundles", "."),
+		e2e.OptAppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
```

Comment:
I'm pretty sure we don't need to source the runtime here, as we're sourcing it in the next command. I think we can safely remove this to cut down on execution time.

Reply:
Let me give it another shot. When I had this line disabled I was running into issues with the tests failing.